src/components/pages/template/edit/field-tree.tsx

```diff
@@ -38,22 +50,184 @@ export default function FieldTree() {
   const generateNodeProps = (row) => ({
     title: row.node.name,
     buttons: [
-      <IconButton
-        minW="auto"
-        aria-label="Edit Field"
-        variant="link"
-        onClick={() => setCurrentField(row.node)}
-        icon={<EditIcon />}
-      />,
-      <IconButton
-        colorScheme="red"
-        minW="auto"
-        aria-label="Delete Field"
-        variant="link"
-        onClick={() => removeFieldById(row.node.fieldId)}
-        icon={<DeleteIcon />}
-      />
-    ]
+      (row.node.type == "TEXT" || row.node.type == "TEXTAREA" || row.node.type == "RICHTEXT") && (
```

Comment:
Instead of using like (row.node.type == "TEXT" ) please use constants that we have already [here](https://github.com/biodiv-platform/cca-ui/blob/master/src/components/pages/data/edit/field-show.tsx)

Reply:
done